crates/database/src/lib.rs

```diff
@@ -69,10 +68,20 @@ pub const ALL_TABLES: &[&str] = &[
     "app_data",
 ];
 
+/// The names of potentially big volume tables we use in the db.
+pub const LARGE_TABLES: &[&str] = &["order_events"];
+
+pub fn all_tables() -> Vec<&'static str> {
+    let mut combined = Vec::with_capacity(TABLES.len() + LARGE_TABLES.len());
+    combined.extend_from_slice(TABLES);
+    combined.extend_from_slice(LARGE_TABLES);
+    combined
+}
```

Comment:
FYI: This function can be written such that it doesn't have to allocate. Obviously doesn't matter here because this function is not performance critical but I guess it's useful to learn about iterator like that.

```suggestion
pub fn all_tables() -> impl Iterator<Item = &'static str> {
    TABLES.iter().copied().chain(LARGE_TABLES.iter().copied())
}
```

Reply:
Thanks, I was looking for that :)